config/jest/jestConfig.js

```diff
@@ -10,6 +11,7 @@ module.exports = {
   testEnvironment: 'jsdom',
   setupFilesAfterEnv: paths.setupTests,
   prettierPath: require.resolve('prettier'),
+  modulePaths: rootResolution ? [cwd()] : undefined,
```

Comment:
Does this work?

```suggestion
  modulePaths: rootResolution ? ['<rootDir>'] : undefined,
```

Reply:
It probably does, but I wanted to keep it consistent with the babel transform root, which we get from the result of `cwd`. I think if you're running `sku test` via your package manger, this will end up being your project root anyway.